fast-cart/src/env.js

```diff
@@ -0,0 +1,6 @@
+const env = {
+    API_URL : 'https://api.bestbuy.com/v1/',
+    API_TOKEN : 'qhqws47nyvgze2mq3qx4jadt',
+};
+
+export default env;
```

Comment:
You can try installing dotenv package and save all the secrets and tokens in .env file

https://www.npmjs.com/package/dotenv